docs: PLT-1397 clarify CDI upload proxy wording for ova import (#4053)
spectrocloud/librarium: docs/docs-content/vm-management/create-manage-vm/advanced-topics/deploy-import-ova.md

    | ------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------- |

>     | **OVA Path**                                | The path to the image your have uploaded to your VM. The path for the example provided is `/root/bitnami-wordpress-6.2.2-r1-debian-11-amd64.ova/`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |                                   |

Reviewer: ```suggestion
    | **OVA Path**                                | The path to the OVA you uploaded to your VM. The path for the example provided is `/root/bitnami-wordpress-6.2.2-r1-debian-11-amd64.ova/`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |                                   |
```

>     | **OVA Path**                                | The path to the image your have uploaded to your VM. The path for the example provided is `/root/bitnami-wordpress-6.2.2-r1-debian-11-amd64.ova/`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |                                   |

Reviewer 2: Suggestion: "The path to the image your have uploaded" -> "The path to the OVA you have uploaded..."
